classes/class-p4-campaigns.php

```diff
@@ -27,14 +44,14 @@ public function __construct() {
 		 * Class hooks.
 		 */
 		private function hooks() {
-			add_action( 'post_tag_add_form_fields',              array( $this, 'add_taxonomy_form_fields' ) );
-			add_action( 'post_tag_edit_form_fields',             array( $this, 'add_taxonomy_form_fields' ) );
-			add_action( 'create_post_tag',                       array( $this, 'save_taxonomy_meta' ) );
-			add_action( 'edit_post_tag',                         array( $this, 'save_taxonomy_meta' ) );
-			add_action( 'admin_enqueue_scripts',                 array( $this, 'enqueue_admin_assets' ) );
+			add_action( 'post_tag_add_form_fields', array( $this, 'add_taxonomy_form_fields' ) );
```

Comment:
Suggestion: Would be nice if you can transform these to short array notation.
eg. `[ $this, 'add_taxonomy_form_fields'  ]`